process/factory/interceptorscontainer/baseInterceptorsContainerFactory.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/ElrondNetwork/elrond-go/sharding"
 )
 
-const numGoRoutines = 2000
+const numGoRoutines = 100
```

Comment:
Dramatic decrease :)

Reply:
Yep,, this was the suggested value by @iulianpascalau